fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/CodeSystemDAOImpl.java

```diff
@@ -54,6 +58,14 @@ protected Connection getConnection() {
         return this.connection;
     }
 
+    /**
+     * Getter for the FHIR data schema
+     * @return
+     */
+    protected String getSchemaName() {
```

Comment:
So this can be picked up in ParameterDaoImpl?

Reply:
Pls explain

Reply:
Why is this here?  Why is it used for?